app/controllers/auth_controller.rb

```diff
@@ -39,20 +37,14 @@ class App < Sinatra::Application
   register_user = lambda do
     email = params['email']
```

Comment:
these variables can be removed and the params directly added to the create method since they are not re-used